app/views/welcome/index.html.erb

```diff
@@ -8,7 +8,7 @@
         <h5 class="card-title">
           Mission</h5>
         <h6 class="card-subtitle mb-2">
-          Find your 'swole mate' and fitness besties by adding your gyms to your profile, adding friends and creating events to go work out.</h6>
+          Find your future 'swolemate' or fitness bestie at your gym and schedule a workout together!</h6>
       </div>
```

Comment:
I updated the mission statement -- let me know what you all think.  We can probably continue to refine this.

My thought here was to make it simpler, while still expressing the purpose of the app.

Reply:
I like it...  apologies for a nit-pick, but in my opinion, would like to go with `SwoleMate`.. I think that's how some of the flash messages (and ReadMe) are set up currently, so while the topic is open, would like to 100% agree on a consistent format for this so we can merge in on clean slate.

Reply:
> I like it... apologies for a nit-pick, but in my opinion, would like to go with `SwoleMate`.. I think that's how most (hopefully all?) of the flash messages (and ReadMe) are set up currently, so while the topic is open, would like to 100% agree on a consistent format for this so we can merge in on clean slate.

@tvaroglu I like that!  And then we can trademark it 😉